instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/TracerProperties.java

```diff
@@ -25,9 +25,10 @@
  */
 @ConfigurationProperties(prefix = "opentelemetry.trace.tracer")
 public final class TracerProperties {
+  // TODO: Add support for Samplers on, off, and probability
```

Comment:
[Note] Make sure to check how OpenTelemetry approaches this in general.

In Dapper, for example, the probability was set to be a single double, so that 0.0 was never, 1.0 was always, and any number in between was a proper probability.

Reply:
I am using this [doc](https://github.com/open-telemetry/opentelemetry-java/blob/master/QUICKSTART.md#sampler) to figure out the properties I need. For the Otel sampler we can set the sampler to be always on, always off, or set a probability from 0 to 1.  I don't think it makes sense for users to set the on/off flags. It makes more sense to just expose the probability.